statistic/memory/memory.go

```diff
@@ -87,8 +82,8 @@ func (u *User) GetIPLimit() int {
 }
 
 func (u *User) AddTraffic(sent, recv int) {
-	u.limiterLock.Lock()
-	defer u.limiterLock.Unlock()
+	u.limiterLock.RLock()
+	defer u.limiterLock.RUnlock()
```

Comment:
Why to use read lock here?

Reply:
This lock is for avoid date race of the pointer of ratelimite,and it's reading the pointer here,not writing.